mail_tracking/models/mail_alias.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, tools
+
+
+class MailAlias(models.Model):
+    _inherit = 'mail.alias'
+
+    @api.model
+    @tools.ormcache('email')
+    def is_email_address_in(self, email):
```

Comment:
The cached method must return the list of aliases strings, not True/False according an email, and do the comparison outside. This way, the cache is not effective, as there can be thousands of possible email comparisons